src/appleseed.studio/mainwindow/renderingsettingswindow.cpp

```diff
@@ -444,17 +444,27 @@ namespace
     };
 
     //
-    // Image Plane Sampling panel.
+    // Image Plane Sampling panels.
     //
 
     class ImagePlaneSamplingPanel
       : public RenderSettingsPanel
+    {
+      public:
+        ImagePlaneSamplingPanel(QWidget* parent = nullptr)
```

Comment:
Should be marked as `explicit`.

Reply:
👍

FWIW Clang-Tidy warns about implicit conversions for other constructors in this file as well, for instance
`RenderSettingsPanel(const QString& title, QWidget* parent = nullptr)`
since it is callable with a single argument.

Reply:
They should be marked as explicit as well. Good catch!

Reply:
Updated.